docs/functions/get_each.md

```diff
@@ -0,0 +1,43 @@
+# get_each
+
+## Description
+Returns an array with the values or a default value in the case it does not exist of each item in a collection
+
+## Parameters
+
+<dl>
+    <dt>key</dt>
+    <dd>Key to search in the collection.</dd>
+    <dt>coll</dt>
+    <dd>Collection where search the expected key.</dd>
+    <dt>default</dt>
+    <dd>Default value to return in the case that the key does not exist (`null` if not provided).</dd>
+</dl>
+
+## Examples
+
+Returns an array with emails from each element of an array, return default value if an element not contain an email:
+```php
+<?php
+
+use function Lambdish\Phunctional\get_each;
+
+$users = [
+    'user1' => [
+        'name'  => 'Mike',
+        'email' => 'mike@example.com'
+    ],
+    'user2' => [
+        'name'  => 'John',
+        'email' => 'john@example.com'
+    ],
+    'user3' => [
+        'name'  => 'James',
+        'phone' => '555-555555'
+    ]
+];
+
+return get_each('email', $users);
+
+// => ['mike@example.com', 'john@example.com', null]
```

Comment:
Could we change this behaviour to "if the key doesn't exists it doesn't return anything".

```suggestion
// => ['mike@example.com', 'john@example.com']
```

I'm thinking in cases and I believe this behaviour would be more useful to people. What do you think?

Reply:
I was thinking about that several times and finally I thought that output array had to have as many elements as input array and put default value in each element which didn't have the key but I am not sure it is the best solution, maybe you are right.

I am going to change the behavior of the function but what would you do with the default value? would you remove it and return and empty array if any element contains the key? I think it would be the best solution.